docs/docs/how-to/logging/log-context.mdx

```diff
@@ -24,75 +24,89 @@ You'll also need to download the sample code for the service:
 
 ```shell
 git clone https://github.com/justtrackio/gosoline.git
-cp -R gosoline/docs/docs/quickstart/http-server/src/write-crud-sql-app
+cp -R gosoline/docs/docs/quickstart/http-server/src/write-crud-sql-app crud-app
 ```
 
 ## Truncate todo text
 
-With this service, users can create, read, update, and delete todos. For the purposes of this tutorial, you'll add some new logic. Instead of accepting any text for a todo, you'll limit the length of that string to prevent users from posting huge amounts of text in their todos. In doing so, you'll store some interesting information in the `Context`. This context information will appear in the logs later.
+With this service, users can create, read, update, and delete todos. For the purposes of this tutorial, you'll add some new logic. Instead of accepting any text for a todo, you'll limit the length of that string to prevent users from posting huge amounts of text in their todos.
 
 In `handler.go`, add a new function:
 
 <CodeBlock title="handler.go" language="go" snippet="truncate">{Handler}</CodeBlock>
 
 In this function, you:
 
-1. Capture the length of the string
-2. Use `log.InitContext()` to initialize special log-related values within the `Context`
-3. Mutate the `Context` to add a log-related field called `original_length`. This stores the length of the string submitted by the user.
+1. Accept a `Context` and a string as arguments
+2. Capture the length of the string
+3. Mutate the `Context` to store the original length of the string
 4. Truncate the string if it is longer than 50 runes.
-5. Returns the potentially truncated string
+5. Return the potentially truncated string
 
-There are two important concepts here:
+For this tutorial, the important thing to pay attention to is where you mutate the `Context`: 
 
-- Log-related context values
-- Mutating context
+```go
+log.MutateContextFields(ctx, map[string]any{
+	"original_length": length,
+})
+```
 
-### Log-related context values
+With Gosoline, you can initilize specific fields that you can use with a `Logger`. (You'll do this in the next step.) Once those fields are initialized, you can append or mutate the fields as you've done here.
 
-When you call `log.InitContext()`, you create two sets of log-related fields in the `Context`:
+:::info
 
-- `localFields`: These fields are limited to the application in which they are set. They are not propagated to downstream services in any way.
-- `globalFields`: These fields aren't limited to the application in which they are set. They are propagated to downstream services.
+Read more about appending and mutating `Context` in our [log package reference](/gosoline/reference/package-log#appendcontextfields).
```

Comment:
Instead of mutating the context, not sure if I would talk about mutating the context fields